DOKS-wordpress/README.md

```diff
@@ -150,6 +152,255 @@ Finally, you need to setup the trusted sources between your MySQL database and y
+  # GIDs are accepted.
+  # The ownership change is carried out recursively down the directory tree.
+  #  GID: ""
+  # The mode value is used to set the file mode of NFS shared directory. Both octals (e.g. 0744)
+  # and incremental/decremental (e.g. "u+r", "o+rw") values are accepted.
+  # The file mode change is carried out recursively down the directory tree.
+  #  mode: ""
+
+  # nfsServerResources defines the NFS server resource requests and limits
+  # Usually, below request and limits are good enough for NFS Server to work
+  # seamlessly(IOs will be taken care by kerner space process i.e nfsd).
+  nfsServerResources: {}
+  #  requests:
+  #    memory: 50Mi
+  #    cpu: 50m
+  #  limits:
+  #    memory: 100Mi
+  #    cpu: 100m
+
+nfsServer:
+  useClusterIP: "true"
+  imagePullSecret: ""
+
+analytics:
+  enabled: "true"
+```
+
+**Note:**
+
+Most of the overrides are self-explanatory and can be customized. Please visit [openebs nfs provisioner helm values](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/deploy/helm/charts/values.yaml) for more details.
+
+Finally, install the chart using Helm:
+
+```console
+helm install openebs-nfs openebs-nfs/nfs-provisioner --version 0.9.0 \
+  --namespace openebs \
+  --create-namespace \
+  -f "openEBS-nfs-provisioner.yaml"
+```
+
+**Note:**
+A specific version for the Helm chart is used. In this case 0.9.0 was picked, which maps to the 0.9.0 version of the application. It’s good practice in general, to lock on a specific version. This helps to have predictable results, and allows versioning control via Git.
+
+You can verify Nginx deployment status via:
+
+```console
+helm ls -n openebs
+```
+
+The output looks similar to (notice that the STATUS column value is deployed):
+
+```text
+NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART                   APP VERSION
+openebs-nfs     openebs         1               2022-05-09 10:58:14.388721 +0300 EEST   deployed        nfs-provisioner-0.9.0   0.9.0  
+```
+
+NFS provisioner requires a block storage device to create the disk capacity required for the NFS server. Here, we will use the default Storage Class (do-block-storage) of the DigitalOcean Kubernetes cluster as the backend storage for the NFS provisioner. In that case, whichever application uses the newly created following Storage Class, can consume shared storage (NFS) on a DigitalOcean volume using OpenEBS NFS provisioner. The following steps create an OpenEBS NFS provisioner supported Storage Class.
+
+Create a YAML file `sc-rwx.yaml`:
+
+```yaml
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: rwx-storage
+  annotations: 
+    openebs.io/cas-type: nsfrwx
+    cas.openebs.io/config: |
+      - name: NSFServerType
+        value: "kernel"
+      - name: BackendStorageClass
+        value: "do-block-storage"
+provisioner: openebs.io/nfsrwx
+reclaimPolicy: Delete
+```
```

Comment:
Usually, we also add some explanations for the relevant fields from the configuration manifest.